docs/tutorial.md

```diff
@@ -9,18 +9,27 @@
 In this tutorial, we'll go through each step of the process to get a basic Discourse deployment.
 
 ## Requirements
+- A working station, e.g., a laptop, with amd64 architecture.
+- Juju 3 installed and bootstrapped to a MicroK8s controller. You can accomplish this process by following this guide: [Set up / Tear down your test environment](https://juju.is/docs/juju/set-up--tear-down-your-test-environment)
+- NGINX Ingress Controller. If you're using [MicroK8s](https://microk8s.io/), this can be done by running the command `microk8s enable ingress`. For more details, see [Addon: Ingress](https://microk8s.io/docs/addon-ingress).
 
-* A laptop or desktop running Ubuntu (or you can use a VM).
-* Juju and [Microk8s](https://juju.is/docs/olm/microk8s) installed. We’ll also want to make sure the ingress add-on is enabled, which we can do by running `microk8s enable ingress`.
+For more information about how to install Juju, see [Get started with Juju](https://juju.is/docs/olm/get-started-with-juju).
 
 ## Steps
 
-### Set up environment
-
-To easily clean up the resources and separate your workload from the contents of this tutorial, set up a new Juju model named `discourse`:
+### Shell into the Multipass VM
 
+To be able to work inside the Multipass VM first you need to log in with the following command:
```

Comment:
This step is confusing since Multipass is not a requirement.

Reply:
Same for other mentions to Multipass VM.

Reply:
We are adding Multipass VM to the tutorials to simplify it for beginners. @erinecon should we add Multipass as a requirement? The tutorial doesn't necessarily need Multipass but we use Multipass in the tutorial to keep it simple to set up.

Reply:
Multipass is set up if the reader follows the instructions in [Set up / Tear down your test environment](https://juju.is/docs/juju/set-up--tear-down-your-test-environment). So I guess that the requirements can be clearer, i.e., instead of "Juju 3 installed and bootstrapped to a MicroK8s controller" the line could read "A Multipass VM with Juju 3 installed and bootstrapped to a MicroK8s controller."

Reply:
Or, if we want to leave the possibility open to **not** use a Multipass VM, then we could update the requirement to say "Juju 3 installed and bootstrapped to a MicroK8s controller. You can accomplish this process by using a Multipass VM as outlined in this guide: [Set up / Tear down your test environment](https://juju.is/docs/juju/set-up--tear-down-your-test-environment)"

Reply:
I agree with the last suggestion so the warnings about Multipass will make sense when reading the tutorial.